HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentView.swift

```diff
@@ -207,8 +213,12 @@ final class PaymentComponentView: UIView {
         selectBankView.addSubview(payInvoiceView)
         selectBankView.addSubview(poweredByGiniView)
         
+        let selectBankViewHeightConstraint = selectBankView.heightAnchor.constraint(equalToConstant: selectBankView.frame.height)
+        selectBankViewHeightConstraint.priority = .required - 1
+        selectBankViewHeightConstraint.isActive = true
```

Comment:
[suggestion]: To have all the constraints activated in the same place we can remove this line
`selectBankViewHeightConstraint.isActive = true` and add `selectBankViewHeightConstraint` in the `NSLayoutConstraint.activate([..])` array .
You can see an example of this in `CameraPreviewViewController` in `setupConstraints()` method -> `cameraFrameViewHeightAnchorPortrait`

Reply:
Same comment for ` Content StackView Constraints`above.